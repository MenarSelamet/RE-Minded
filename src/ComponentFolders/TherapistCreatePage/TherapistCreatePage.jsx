import React from 'react'

function TherapistCreatePage() {
  return (
    <div>TherapistCreatePage</div>
  )
}

export default TherapistCreatePage