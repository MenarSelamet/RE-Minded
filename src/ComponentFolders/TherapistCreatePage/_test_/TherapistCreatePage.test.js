import React from 'react';
import renderer from 'react-test-renderer';
import TherapistCreatePage from '../TherapistCreatePage';

test('Requirements Page snapshot test', () => {
  const component = renderer.create(<TherapistCreatePage />);

  const tree = component.toJSON();

  expect(tree).toMatchSnapshot();
});