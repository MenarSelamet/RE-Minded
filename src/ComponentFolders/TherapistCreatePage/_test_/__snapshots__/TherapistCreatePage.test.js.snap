// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Requirements Page snapshot test 1`] = `
<div>
  TherapistCreatePage
</div>
`;
